stdlib/src/python/object.mojo

```diff
@@ -119,6 +119,14 @@ struct PythonObject(
         """Initialize the object with a `None` value."""
         self.__init__(None)
 
+    fn __init__(inout self, *, other: Self):
```

Comment:
**Nit** I think you accidentally omitted the explicit conformance to `CollectionElementNew` for `PythonObject`—you added it for the other types.

Not a major problem, I'll sync this in anyway. But wouldn't hurt if you wanted to drop that in as an add-on aside to any future PRs you do 🙂

Reply:
Yeah I tried, but I hit a compiler bug when doing so. It's hard to do a minimal repro. It's really weird

Reply:
Huh, that does sound odd. Even if you can't get a minimal repro, if you can trigger it again easily or see it again, can you put it in a comment here? Then I'll at least know if I run into it that it's the same thing you saw.